src/patterns/error-status-pages/index.njk

```diff
@@ -164,27 +164,36 @@ The page should have:
     patternlibExample({ "path": "patterns/error-status-pages/examples/503-error/index.njk" })
 }}
 
-## Timeout
+## 408 timeout error
```

Comment:
```suggestion
## Session timed out error
```

...after a discussion with the devs, 408 is not the correct code, and in the next round of changes on this pattern, we'll remove all other references to http codes